config/environment.js

```diff
@@ -47,26 +46,26 @@ module.exports = function (environment) {
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true
     // ENV.APP.LOG_VIEW_LOOKUPS = true
     ENV.enableChallengeCreator = true
-    ENV.pbAnalyticsApi.baseURL = 'http://localhost:3000'
     ENV.pbApi.baseURL = 'http://localhost:3006'
   }
 
   if (environment === 'test') {
     // Testem prefers this...
     ENV.baseURL = '/'
     ENV.locationType = 'none'
-    ENV.APP.autoboot = false // <------ false here
+    ENV.APP.autoboot = false // <------ false here (?)
```

Comment:
JAJAJAJ no encontré explicaciones directas, pero por lo que dice [acá](https://hackernoon.com/how-does-ember-boot-5e1f9e7a1117) y [acá](https://api.emberjs.com/ember/release/classes/Application/methods) autoboot parece ser un feature que tiene que estar prendido casi siempre. Y en ember qunit [dice](https://github.com/emberjs/ember-qunit/blob/master/README.md) que debería estar en false.

Así que un mejor comment podría ser:

```suggestion
    ENV.APP.autoboot = false // ember-qunit needs it to be false.
```